python/cuml/fil/fil.pyx

```diff
@@ -574,6 +575,15 @@ class ForestInference(Base):
             model passed.
 
         """
+        if(has_treelite() is False):
+            raise ImportError("Treelite needs to be built from source"
```

Comment:
Philip will update treelite so we just need to specify the minimum version. Should not need build from source

Reply:
i thought i should add it in, just in case there is some issue while creating the new pip version

Reply:
I'd prefer to be more general, since building from source is less likely. Just like "you need to install 0.32" should be clear.